connections/ethereum/connection_test.go

```diff
@@ -97,3 +98,22 @@ func TestConnection_SafeEstimateGasMax(t *testing.T) {
 		t.Fatalf("Gas price should equal max. Suggested: %s Max: %s", price.String(), maxPrice.String())
 	}
 }
+
+func TestConnection_SafeEstimateGasMin(t *testing.T) {
+	minPrice := big.NewInt(1)
+	conn := NewConnection(TestEndpoint, false, AliceKp, log15.Root(), GasLimit, MaxGasPrice, minPrice, big.NewFloat(0), "", "")
+	err := conn.Connect()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer conn.Close()
+
+	price, err := conn.SafeEstimateGas(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if price.Cmp(minPrice) != 0 {
```

Comment:
This test passes because the price is greater than the `minPrice` and thus the `Cmp` statement is not going to be equal to zero. I'm thinking an additional test where the price does get set to the minimum (as one of the tests but for the maximum gas price does above) would be helpful for full scope of the functionality.

Reply:
@vezenovm 

Thanks for your review.
NewFloat(0)` for the `GasMultipler` argument of NewConnection in TestConnection_SafeEstimateGasMin, so I think that the gasPrice without considering minGasPrice will be 0. I think.
Therefore, since it is lower than minGasPrice(`big.NewFloat(1)`), the final price will be 1, and we believe that `price.Cmp(minPrice)` will be 0, which is the same.

Maybe it is hard to see that `GasMultipler` is 0, so I think I will make it a local variable, how about that?

Reply:
I've added a test, what do you think?
https://github.com/ChainSafe/ChainBridge/pull/689/commits/29e0f890d3b47190f2282de7ac84290153ea65c5

Reply:
Apologies for getting back late. I did miss that you made the `GasMultiplier` 0, and your logic is correct there. I think the local variable addition does help make it more clear as well.

I also think the second test is still good to have as it shows the logic for when the `GasMultiplier` is 1 which is the more common case.